Utilities/bootstrap

```diff
@@ -542,7 +542,7 @@ def main():
         args.swiftc_path = get_swiftc_path()
 
     build_actions = set(args.build_actions)
-            
+
```

Comment:
Please drop spurious whitespace changes, if possible. Not a biggie, but makes for more readable diffs.